src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj.smartdashboard.*;
+
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.controls.VelocityVoltage;
+
+public class Shooter extends SubsystemBase {
+
+  private final TalonFX pivotMotor = new TalonFX(0); // TODO: Set to pivot motor device Id
+  private final TalonFX leftShootMotor = new TalonFX(0); // TODO: Set to left motor device Id
+  private final TalonFX rightShootMotor = new TalonFX(0); // TODO: Set to right motor device Id
+  private double speed = 0;
+  private VelocityVoltage velocity = new VelocityVoltage(speed);
+  public static boolean motorsAtShootingSpeed = false;
```

Comment:
This shouldn't be public. The only fields that should be public is fields that you are completely fine with other code modifying and this isn't one of those pieces.